ui-ngx/src/app/shared/components/file-input.component.html

```diff
@@ -25,13 +25,13 @@
                 [flowConfig]="{allowDuplicateUploads: true}">
     <div class="tb-file-select-container">
       <div class="tb-file-clear-container">
-        <button mat-icon-button color="primary"
+        <button mat-icon-button
                 type="button"
                 (click)="clearFile()"
                 class="tb-file-clear-btn"
                 matTooltip="{{ 'action.remove' | translate }}"
                 matTooltipPosition="above">
-          <mat-icon>close</mat-icon>
+          <mat-icon>delete</mat-icon>
```

Comment:
Changing this file affects the entire platform. You cannot change the default icon on the platform. You need to add the ability to change the icon via the input in the component.